example/flutter/flutter_example.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016, the Flutter project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:usage/usage.dart';
+
+import 'package:flutter/material.dart';
+
+import 'dart:io';
+
+void main() {
+  print("${Platform.environment}");
+  runApp(
+    new MaterialApp(
+      title: 'Usage Example',
+      theme: new ThemeData.dark(),
+      routes: <String, RouteBuilder>{
+        '/': (RouteArguments args) => new FlutterDemo()
+      }
+    )
+  );
+}
+
+class FlutterDemo extends StatefulComponent {
+  State createState() => new _FlutterDemoState();
+}
+
+class _FlutterDemoState extends State<FlutterDemo> {
+  Analytics _ga;
+  int _times;
+
+  @override
+  void initState() {
+    _times = 0;
+    _ga = new Analytics('UA-67589403-4', 'ga_test', '1.0')
+      ..optIn = true
+      ..sendScreenView('home');
+    super.initState();
+  }
+
+  void _handleButtonPressed(Size newSize) {
+    _ga.sendEvent('button', 'pressed');
+    setState(() {
+      _times++;
+    });
+  }
+
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      toolBar: new ToolBar(
+        center: new Text('Usage Example')
+      ),
+      body: new Center(
+        child: new Text("Button pressed $_times times.")
+      ),
+      floatingActionButton: new FloatingActionButton(
+        child: new Icon(
+          icon: 'content/add'
```

Comment:
Yeah, I'm updating this example to demonstrate the opt-in feature, will fix that as well.